src/components/User/UserInfo.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect } from "react"
+import PropTypes from "prop-types"
+
+import styles from "./UserInfo.module.css"
+
+import useHttp from "../../hooks/use-http"
+import { getUserInfo } from "../../lib/api"
+
+import LoadingSpinner from "../UI/LoadingSpinner"
+import Stats from "./Stats"
+import Avatar from "../UI/Avatar"
+import Error from "../UI/Error"
+
+const UserInfo = ({ userId }) => {
+  const {
+    sendRequest,
+    status,
+    data: userInfo,
```

Comment:
і компонент UserInfo, і дані userInfo
в return `userInfo` написано 5 разів. цього можна легко уникнути

```suggestion
    data: { stats, user },
```

Reply:
звучало красиво, но нужно еще подумать